tf_encrypted/keras/engine/base_layer.py

```diff
@@ -97,6 +99,12 @@ def __call__(self, inputs, *args, **kargs):
 
     return outputs
 
+  def add_weight(self, variable):
```

Comment:
does it make sense if the input here is always a kernel?

Reply:
I'd say no, for example there are weights in batchnorm that are not kernels